website/_includes/donate-modal.html

```diff
@@ -2,39 +2,116 @@
+      <line x1="18" y1="6" x2="6" y2="18" />
+      <line x1="6" y1="6" x2="18" y2="18" />
+    </svg>
+  </div>
+  <h1 class="text-blue-dark text-center font-4xl font-light">
+    {{ _('Pay What You Want') }}
+  </h1>
   <form action="">
-  {% if LANG in settings.LOCALE_CURRENCIES %}
-    {% set this_currency = settings.LOCALE_CURRENCIES[LANG] %}
-  {% else %}
-    {% set this_currency = 'usd' %}
-  {% endif %}
-  {% set this_curdata = settings.CURRENCIES[this_currency] %}
-    <p>Select an amount:</p>
-    <select name="currency-switcher-currency" id="id_currency-switcher-currency">
-    {% for currency, vals in settings.CURRENCIES | dictsort %}
-      <option value="{{ currency }}" {% if currency == this_currency %}selected{% endif %}>{{ currency.upper() }}   {{ vals.symbol.decode('utf-8') }}</option>
-    {% endfor %}
-    </select>
-    <input type="radio" id="amt1" name="amount" value="{{ this_curdata.presets[0] }}">
-    <label for="amt1">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.presets[0] }}</label><br>
-    <input type="radio" id="amt2" name="amount" value="{{ this_curdata.default }}" checked>
-    <label for="amt2">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.default }}</label><br>
-    <input type="radio" id="amt3" name="amount" value="{{ this_curdata.presets[2] }}">
-    <label for="amt3">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.presets[2] }}</label><br>
-    <input type="radio" id="amt4" name="amount" value="{{ this_curdata.presets[3] }}">
-    <label for="amt4">{{ this_curdata.symbol.decode('utf-8') }}{{ this_curdata.presets[3] }}</label><br><br>
-    <input type="radio" name="amount" value="">{{ _('Other') }}<input type="text" name="amount" />
-    <button id="amount-submit">{{ _('Donate & Download') }}</button>
-    <button id="amount-cancel">{{ _('No Thanks; Close & Download') }}</button>
+    <section class="font-lg pt-0 pl-6 pb-4 pr-6 leading-normal text-center">
+      <p>{{ _('Thunderbird is independent, free, and Open Source software sustained by your donations. Help us keep the project alive!') }}</p>
+    </section>
+    <section class="bg-grey-light pt-4 pl-6 pb-4 pr-6">
+      <div class="flex justify-between items-center">
+        <label class="uppercase font-sm">{{ _('Select an amount') }}</label>
+        <label>
+          {{ _('Currency') }}
+          <select id="id_currency-switcher" class="inline-block w-24 bg-white p-1 border border-solid border-grey" name="currency-switcher">
+            {% for currency, vals in settings.CURRENCIES | dictsort %}
+            <option value="{{ currency }}" {% if currency == this_currency %}selected{% endif %}>{{ currency.upper() }}   {{ vals.symbol.decode('utf-8') }}</option>
+            {% endfor %}
+          </select>
+        </label>
+      </div>
+      <div id="amount-selection" class="amount-selection flex flex-wrap mt-3">
+        <label class="flex items-center shadow-md mb-2 mr-4">
```

Comment:
We can add a `rounded` class to all this value selectors. The last one with the amount needs an `overflow: hidden` so the input field inside doesn't bleed out.